src/scripts/pokemons/PokemonList.ts

```diff
@@ -923,6 +1204,25 @@ const pokemonList = createPokemonArray(
             'speed': 95,
         },
     },
+    {
+        'id': 50.01,
+        'name': 'Alolan Diglett',
+        'nativeRegion': GameConstants.Region.alola,
+        'catchRate': 255,
+        'type': [PokemonType.Ground, PokemonType.Steel],
+        'levelType': LevelType.mediumfast,
+        'exp': 53,
+        'eggCycles': 20,
+        'evolutions': [new LevelEvolution('Alolan Diglett', 'Alolan Dugtrio', 26)],
```

Comment:
Just double checking that these level evolutions can happen anywhere as they are already in their Alolan form right?

Reply:
I can't think of a problem with alolan --> alolan evolutions being unrestricted.